onmt/modules/multi_headed_attn.py

```diff
@@ -142,13 +173,18 @@ def __init__(self, head_count: int, model_dim: int, dropout: float = 0.1,
                 vocab_size, self.dim_per_head)
         else:
             self.relative_positions_embeddings = None
+            if max_relative_positions == -1:
+                self.freqs_cis = precompute_freqs_cis(
+                    self.dim_per_head,
+                    512 * 2)  # 512 = maxseqlen
```

Comment:
maxseqlen for what? llama pretrained models? is it a hard limit or just a good practice/value from other work?